SECURITY.md

```diff
@@ -0,0 +1,99 @@
+# Security Policy for the Crypt::OpenPGP distribution.
+
+Report issues via email at: Timothy Legge <timlegge@gmail.com>.
+
+This is the Security Policy for the Crypt::OpenPGP distribution.
+
+The latest version of this Security Policy can be found on the
+Crypt::OpenPGP website at
+https://github.com/perl-Crypt-OpenPGP/Crypt-OpenPGP/SECURITY.md
+
+This text is based on the CPAN Security Group's Guidelines for Adding
+a Security Policy to Perl Distributions (version 0.1.8)
+https://security.metacpan.org/docs/guides/security-policy-for-authors.html
+
+# How to Report a Security Vulnerability
+
+Security vulnerabilities can be reported by e-mail to the current
+project maintainers at Timothy Legge <timlegge@gmail.com>
+
+Please include as many details as possible, including code samples
+or test cases, so that we can reproduce the issue.
+
+If you would like any help with triaging the issue, or if the issue
+is being actively exploited, please copy the report to the CPAN
+Security Group (CPANSec) at <cpan-security@security.metacpan.org>.
+
+Please *do not* use the public issue reporting system on RT or
+GitHub issues for reporting security vulnerabilities.
+
+Please do not disclose the security vulnerability in public forums
+until past any proposed date for public disclosure, or it has been
+made public by the maintainers or CPANSec.  That includes patches or
+pull requests.
+
+For more information, see
+[Report a Security Issue](https://security.metacpan.org/docs/report.html)
+on the CPANSec website.
+
+## Response to Reports
+
+The maintainer(s) aim to acknowledge your security report as soon as
+possible.  However, this project is maintained by a single person in
+their spare time, and they cannot guarantee a rapid response.  If you
+have not received a response from them within 7 days, then
+please send a reminder to them and copy the report to CPANSec at
+<cpan-security@security.metacpan.org>.
+
+Please note that the initial response to your report will be an
+acknowledgement, with a possible query for more information.  It
+will not necessarily include any fixes for the issue.
+
+The project maintainer(s) may forward this issue to the security
+contacts for other projects where we believe it is relevant.  This
+may include embedded libraries, system libraries, prerequisite
+modules or downstream software that uses this software.
+
+They may also forward this issue to CPANSec.
+
+# Which Software this Policy Applies to
```

Comment:
Trivial nit: Using standard title capitalization rules, this line should be

`# Which Software This Policy Applies To`

Please refer to https://capitalizemytitle.com/, which will capitalize a title using 9 different commonly used standard guides. In this case, they all agree as to how this should be capitalized.

Reply:
Thanks we have added that to https://github.com/CPAN-Security/security.metacpan.org/pull/119 and I have updated to the latest version of that document here